packages/react/src/components/UIShell/components/SideNavDecorativeIcon.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright IBM Corp. 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+import cx from 'classnames';
+import { Menu, Pin } from '@carbon/icons-react';
+import { usePrefix } from '../internal/usePrefix';
+
+export default function SideNavDecorativeIcon() {
```

Comment:
```suggestion
export default function SideNavPin() {
```

Should we name this SideNavPin for the future plans for this?